.github/workflows/ecosystem.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - master
-  pull_request:
+  #pull_request:
```

Comment:
No longer running ecosystem on PRs, only on push, makes sense to me, but is this intentional?

Reply:
this i'll revert. I wanted to minimize CI since it was really only the Win Azure lane failing.